python/console/console_output.py

```diff
@@ -85,6 +81,34 @@ def isatty(self):
         return False
 
 
+FULL_HELP_TEXT = QCoreApplication.translate("PythonConsole", """QGIS Python Console
+======================================
+
+The console is a Python interpreter that allows you to execute python commands
+QGIS libraries are preloaded and can be used directly from the console.
+
+Useful variables:
+
+- iface.mainWindow() will return the Qt Main Window
```

Comment:
```suggestion
- iface.mainWindow() will return the Qt Main Window.
```

Reply:
I'd rather remove the dot from all entries

Reply:
:+1: for no dots from me